packages/tabs/examples/default-tabs.md

```diff
@@ -70,7 +77,7 @@ initialState = {
       >
```

Comment:
Probably not related to this PR, but I noticed a React warning in the Advanced Usage example:

![rendering-pure](https://user-images.githubusercontent.com/1811365/68710250-09e6de80-054c-11ea-8116-739e978d4aca.gif)

I wasn't able to pinpoint the impure part of the render yet, but will keep looking into it.

Reply:
This is visible often in our `stylguidist` environment. Not related to this PR